test/prog/dftb+/timedep/C4H6-S1-Force_uncache-Stratmann/XREST.DAT

```diff
@@ -0,0 +1,11 @@
+Mulliken analysis of excited state S 1
```

Comment:
Any reason to check this file in, as its not used by the regression testing? (Likewise for other .DAT files)

Reply:
No, happened by mistake. Will remove it.